src/shared/schemas.ts

```diff
@@ -33,7 +33,7 @@ export type SchemaType = 'yaml' | 'json'
 export interface SchemaMapping {
     uri: vscode.Uri
     type: SchemaType
-    owner?: string
+    registry?: string
```

Comment:
I see a lot of early returns due to this being nullable. This field is required according to https://github.com/aws/aws-toolkit-vscode/pull/3058/files#diff-6d14e8ae512380b5cec4abff41a0948289247dd6ebb53126cea22406bacbe7b0R94
```suggestion
    registry: string
```